vendor/engines/split_accounts/app/views/facility_accounts/account_fields/split_accounts/_splits.html.haml

```diff
@@ -0,0 +1,17 @@
+- available_subaccounts = Account.excluding_split_accounts.filter_global_account_types
```

Comment:
Yeah, I wasn't a fan of putting this in the view either.  Didn't get around to cleaning it up yet.  Good idea.